src/components/DataExplorerFrame.js

```diff
@@ -0,0 +1,34 @@
+import { iframeResizer } from 'iframe-resizer'
+import _ from 'lodash/fp'
+import { useLayoutEffect, useRef } from 'react'
+import { h, iframe } from 'react-hyperscript-helpers'
+import datasets from 'src/data/datasets'
+import * as Nav from 'src/libs/nav'
+
+
+const DataExplorerFrame = ({ dataset }) => {
+  const elem = useRef()
+  const { origin } = _.find({ name: dataset }, datasets)
+
+  useLayoutEffect(() => {
```

Comment:
Does this need to be `useLayoutEffect`? The docs recommend using `useEffect` if possible.